sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/interpreter/operator/BeamSqlCaseExpression.java

```diff
@@ -51,7 +51,8 @@ public BeamSqlCaseExpression(List<BeamSqlExpression> operands) {
 
   @Override public BeamSqlPrimitive evaluate(BeamRecord inputRow, BoundedWindow window) {
     for (int i = 0; i < operands.size() - 1; i += 2) {
-      if (opValueEvaluated(i, inputRow, window)) {
+      Boolean b = opValueEvaluated(i, inputRow, window);
```

Comment:
nit: `b` -> `wasOpEvaluated`

Reply:
`opValueEvaluated()` is not necessarily boolean, it's a function to execute the expression and extract a result from it. It's basically `executeExpressionAndGetValue(expressionIndex, input, window)`